libs/core/src/lib/form/form-message/consts.ts

```diff
@@ -0,0 +1,26 @@
+import { MessageStates } from '@fundamental-ngx/core';
```

Comment:
not a big deal, but this file should be named `constants.ts` for consistency with the other components.